playbooks/tasks/iworx-php-scl.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+- name: Enable Interworx SCL PHP Support
+  command: >
+    nodeworx
+    -un
+    -c Http
+    -a enableMultiplePhp
+
+- name: Enable PHP Versions + Set Default Version
+  command: >
+    nodeworx
+    -un
+    -c Http
+    -a multiplePhpOptions
+    --enabled_php_versions /opt/remi/php56,/opt/remi/php70,/opt/remi/php71
```

Comment:
You need to auto-detect these.  I would just look for /opt/remi/php* and put them here.